src/components/cart/EmptyCart/EmptyCart.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react'
 import Button from 'src/components/ui/Button'
 import { ShoppingCart as ShoppingCartIcon } from 'phosphor-react'
-import { useCartToggleButton } from 'src/sdk/cart/useCartToggleButton'
 
 import './empty-cart.scss'
 
-function EmptyCart() {
-  const toggleProps = useCartToggleButton()
+interface Props {
+  /**
+   * This function is called when `Start Shopping` button is clicked
+   */
+  onDismiss: () => void
```

Comment:
Naming it `onDismiss` because it what it does now. But we can change it for something more generic.